modules/configuring-vsphere-connection-settings.adoc

```diff
@@ -25,6 +26,13 @@ Modify the following vSphere configuration settings as required:
 . In the Administrator perspective, navigate to *Home -> Overview*.
 . Under *Status*, click *vSphere connection* to open the *vSphere connection configuration* wizard.
 . In the *vCenter* field, enter the network address of the vSphere vCenter server. This can be either a domain name or an IP address. It appears in the vSphere web client URL; for example `https://[your_vCenter_address]/ui`. 
+. In the *vCenter cluster* field, enter the name of the vSphere vCenter cluster where {product-title} is installed.
++
+[NOTE]
+====
+This step is mandatory for {product-title} 4.13 installations and later.
```

Comment:
This suggestion roughly follows the IBM style guide's examples for [References to earlier or later versions](https://www.ibm.com/docs/en/ibm-style?topic=references-product-versions#references-to-earlier-or-later-versions)

```suggestion
This step is mandatory for {product-title} 4.13 or later.
```